### PR TITLE
docs: Added kata-runtime to implementations

### PR DESCRIPTION
**Pull request by jodh-intel**
Added the Kata Containers project runtime to the list of
implementations.

Signed-off-by: James O. D. Hunt <james.o.hunt@intel.com>

### CODE REVIEW

File: implementations.md [hunk 1]
```diff
@@ -13,6 +13,8 @@ If you know of any associated projects that are not listed here, please file a p
 
 * [hyperhq/runv][runv] - Hypervisor-based runtime for OCI
 * [clearcontainers/runtime][cc-runtime] - Hypervisor-based OCI runtime utilising [virtcontainers][virtcontainers] by Intel®.
+* [kata-containers/runtime][kata-runtime] - Hypervisor-based OCI runtime
+  combining technology from [clearcontainers/runtime][cc-runtime] and [hyperhq/runv][runv].
```
Comment (tianon) [inline implementations.md]: The line-wrap here doesn't seem quite right -- I think this should all be one line per https://github.com/opencontainers/runtime-spec/blob/master/style.md#one-sentence-per-line

**Comment by jodh-intel** [comment]
Thanks for reviewing. Branch updated.

**Comment by caniszczyk** [comment]
LGTM

**Comment by jodh-intel** [comment]
Now that [Kata Containers has been released](https://katacontainers.io/posts/kata-containers-first-release/), it would be great if we could get this landed so folk are aware of it :)

**Comment by dqminh** [comment]
LGTM

**Pull request merged**